src/cli.rs

```diff
@@ -197,6 +197,10 @@ pub struct ProductOperatorRun {
     /// Tracing log collector system
     #[arg(long, env, default_value_t, value_enum)]
     pub tracing_target: TracingTarget,
+    /// The full name of the operator image. This env var is set by the helm-charts and should
```

Comment:
Nit: Please have one empty newline as spacing between all this struct fields

Reply:
Yes, I prefer that too, but followed what was already done

Updated in b3ed2a06a5afa6b016915d462850ed3d93edab61

Reply:
Thanks!
He have decided on the spaces in an Arch meeting and documented it in our code style guideline (same sheet as arch meeting agenda) :)